gads-ui/src/hooks/useAdmin.js

```diff
@@ -0,0 +1,186 @@
+import { useState } from 'react'
+
+import { api } from '../services/api'
+
+export function useAdmin() {
+    const [providers, setProviders] = useState([])
+
+    const listProviders = async () => {
+        try{
+            const response = await api.get(`/admin/providers`)
+
+            if(response.status === 200) {
+                let providers = response.data ?? {}
```

Comment:
```suggestion
                let providers = response.data ?? []
```
Shouldn't it be an array like the default `useState()` value?